cypress/fixtures/scoreActions/disqualifyingEntities.json

```diff
@@ -0,0 +1,446 @@
+[
```

Comment:
These seem like it data for the tests. The fixtures are supposed to be fixed responses for network requests, or other operations like file loading which we use to fake the `.cl` import.

Reply:
Yes this is data for the tests. The FIXTURES folder for it seems like an appropriate place to me. I created a sub folder to keep these separate from the .cl files.

According to the Cypress documentation at https://docs.cypress.io/api/commands/fixture.html fixtures are: "a fixed set of data located in a file."